src/GafferArnold/IECoreArnoldPreview/ShaderNetworkAlgo.cpp

```diff
@@ -337,10 +339,10 @@ namespace ShaderNetworkAlgo
 std::vector<AtNode *> convert( const IECoreScene::ShaderNetwork *shaderNetwork, const std::string &name, const AtNode *parentNode )
 {
 	ShaderNetworkPtr networkCopy;
-	if( true ) // todo : make conditional on OSL < 1.10
+	if( OSL_VERSION < 11000 )
```

Comment:
I _think_ I'm right in saying that it's not our `OSL_VERSION` that matters here, rather it's the OSL version that Arnold is using internally. All Arnold 6 versions seem to be on 1.11, and so does 5.4.0.2 (I haven't looked further back). So I think we may be able to rip the call to `convertOSLComponentConnections()` out completely. Could you give that a try?

Reply:
I confirmed just removing this works fine when building Gaffer with older OIIO. Addressed in 22a627dc4f68de1479127bc606909f2d30b44f90